src/components/controls/dates/Calendar/Calendar.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import { expect } from "@storybook/jest"
+import { ComponentMeta, ComponentStory } from "@storybook/react"
+import { userEvent, within } from "@storybook/testing-library"
+import { useState } from "react"
+import { Calendar } from "./Calendar"
+
+export default {
+  title: "Controls/Dates/Calendar",
+  component: Calendar,
+  argTypes: {
+    disabled: {
+      control: "boolean",
+    },
+  },
+} as ComponentMeta<typeof Calendar>
+
+const Template: ComponentStory<typeof Calendar> = (args) => {
+  const { value: defaultValue } = args
+  const [value, setValue] = useState<Date | null>(defaultValue || null)
```

Comment:
What do you think about inferring types like this?

```suggestion
  const [value, setValue] = useState(defaultValue)
```